.github/workflows/deployTraining.yml

```diff
@@ -50,11 +50,16 @@ jobs:
       - name: Install dependencies
         run: yarn install
       - name: Build deploy artifact
+        env:
+          REACT_APP_BASE_URL: https://training.simplereport.gov
+          REACT_APP_BACKEND_URL: https://training.simplereport.gov/api
+          REACT_APP_IS_TRAINING_SITE: true
+          PUBLIC_URL: /app/
         run: yarn run build
       - uses: azure/login@v1
         with:
           creds: ${{ secrets.AZURE_CREDENTIALS }}
       - name: Deploy frontend app
-        run: yarn run deploy-azure-${{ env.DEPLOY_ENV }}
+        run: az storage blob upload-batch -s build/ -d '$web' --account-name simplereporttrainingapp --destination-path '/app'
```

Comment:
If we change this one more time, it might be time to have a Makefile for all our `az` commands that vary by, like, one short interpolated string. 😁

Reply:
This is probably what we should do